browser/components/MarkdownPreview.js

```diff
@@ -125,6 +125,12 @@ body p {
   .clipboardButton {
     display: none
   }
+  pre.CodeMirror>code {
+    white-space: pre-wrap;
```

Comment:
please add `word-wrap: break-word;`.
Now, output PDF is broken like below:
<img width="1152" alt="2018-06-25 17 06 00" src="https://user-images.githubusercontent.com/14838850/41837992-1394a592-789a-11e8-95ef-cdf39f2e021f.png">